src/index.html

```diff
@@ -22,6 +22,376 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#index.html"
+        class="header__logo-link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo image"
+          class="header__logo"
+        />
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a
+              href="#1"
+              class="header__nav-link is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#1"
+              class="header__nav-link"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#1"
+              class="header__nav-link"
+            >
+              smartphones
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#1"
+              class="header__nav-link"
+              data-qa="nav-hover"
+            >
+              Laptops &amp Computers
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#1"
+              class="header__nav-link"
+            >
+              gadgets
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#1"
+              class="header__nav-link"
+            >
+              tablets
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#1"
+              class="header__nav-link"
+            >
+              photo
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#1"
+              class="header__nav-link"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div
+        class="product-card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="imac image"
+          class="product-card__image"
+        />
+        <p class="product-card__title">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="product-card__code">Product code: 195434</p>
+        <div class="product-card__reviews">
+          <div class="product-card__reviews-stars stars">
+            <div class="star"></div>
+            <div class="star"></div>
+            <div class="star"></div>
+            <div class="star"></div>
+            <div class="star"></div>
+          </div>
```

Comment:
```suggestion
          <div class="product-card__reviews-stars stars stars--4">
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
          </div>
```
Consider using the HTML structure from the previous task